### PR TITLE
Fixed Polling Dao

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-local-java-cdc-connector-polling/src/main/java/io/eventuate/local/polling/PollingDao.java [hunk 1]
```diff
@@ -125,25 +115,45 @@ public int processEvents(BinlogEntryHandler handler) {
...
 
     return ids.size();
   }
 
-  private void handleEvent(Object id, BinlogEntryHandler handler, SqlRowSet sqlRowSet) {
+  private void markEventsAsProcessed(List<CompletableFuture<Object>> futureIds, String pk, BinlogEntryHandler handler) {
+    List<Object> ids = futureIds
+            .stream()
+            .map(futureId -> {
+              try {
```
Comment (cer) [inline eventuate-local-java-cdc-connector-polling/src/main/java/io/eventuate/local/polling/PollingDao.java]: Also, how many places in the CDC repeat this code pattern: cf.get() wrapped in a try/catch that throws. Define a static helper function,. CompleteFutureUtil.get(cf)

Comment (dartartem) [reply eventuate-local-java-cdc-connector-polling/src/main/java/io/eventuate/local/polling/PollingDao.java]: ok, will do

File: build.gradle [hunk 2]
```diff
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
 
         maven {
-            url "http://repo.spring.io/milestone/"
+            url "https://repo.spring.io/milestone/"
```
Comment (cer) [inline build.gradle]: Is this repo used?

Comment (dartartem) [reply build.gradle]: it should be, Idea tried to download something from it. I will check.

**Pull request merged**